GLOSSARY.md

```diff
@@ -1,85 +1,113 @@
 # PyTorch Glossary
 
+Several terms in this glossary are used frequently throughout PyTorch documentation and warrant basic definitions.
+
 <!-- toc -->
 
-- [Operation and Kernel](#operation-and-kernel)
-  - [ATen](#aten)
-  - [Operation](#operation)
-  - [Native Operation](#native-operation)
-  - [Custom Operation](#custom-operation)
-  - [Kernel](#kernel)
-  - [Compound Operation](#compound-operation)
-  - [Composite Operation](#composite-operation)
-  - [Non-Leaf Operation](#non-leaf-operation)
-  - [Leaf Operation](#leaf-operation)
-  - [Device Kernel](#device-kernel)
-  - [Compound Kernel](#compound-kernel)
-- [JIT Compilation](#jit-compilation)
-  - [JIT](#jit)
-  - [TorchScript](#torchscript)
-  - [Tracing](#tracing)
-  - [Scripting](#scripting)
+- [PyTorch Glossary](#pytorch-glossary)
+  - [Model, Tensor, Operation and Kernel](#model-tensor-operation-and-kernel)
+    - [Model](#model)
+    - [Tensor](#tensor)
+    - [ATen](#aten)
+    - [Operation](#operation)
+    - [Native Operation](#native-operation)
+    - [Custom Operation](#custom-operation)
+    - [Kernel](#kernel)
+    - [Compound Operation](#compound-operation)
+    - [Composite Operation](#composite-operation)
+    - [Non-Leaf Operation](#non-leaf-operation)
+    - [Leaf Operation](#leaf-operation)
+    - [Device Kernel](#device-kernel)
+    - [Compound Kernel](#compound-kernel)
+  - [JIT Compilation](#jit-compilation)
+    - [JIT](#jit)
+    - [TorchScript](#torchscript)
+    - [Tracing](#tracing)
+    - [Scripting](#scripting)
 
 <!-- tocstop -->
 
-# Operation and Kernel
+## Model, Tensor, Operation and Kernel
+
+### Model
+
+An algorithm for making predictions from data. (E.g. AI Model)
+
+### Tensor
+
+(1:formal) A multilinear map, in the form of a specialized storage data structure, similar to arrays and matrices.
+(2:informal) A multi-dimensional array/matrix containing elements (data) of a single data type (int, float, ...).
+(3:PyTorch implementation) A low-level representation of the specialized storage data structure, containing pointers to storage where the data and metadata are located.
+
+- In PyTorch tensors are used to encode the data inputs & outputs of a model and the model’s parameters.
+
+### ATen
+
+Short for "A Tensor Library." The foundational (low-level) tensor and mathematical
```

Comment:
"A Tensor Library." -> "A Tensor Library".
It also include the [new  C++ Aten bindings ](https://github.com/pytorch/pytorch/pull/105459/files#diff-23289e5bee4b2f3419fc93c14929e42f2bb52e298cec134917c318ae13a9290d)

Reply:
period goes on the inside of the quotes or parens (if considered a sentence) otherwise outside :-)  or is that a split between American and British grammar? my intention was sentence :-O

Reply:
thoughts?